website/addons/figshare/README.md

```diff
@@ -18,4 +18,3 @@ akin to *{"error": "You cannot request an access token for yourself!"}*
   1. Copy the *consumer_key* to **CLIENT_ID**
   2. Copy the *consumer_secret* to **CLIENT_SECRET**
 12. Open website/settings/local.py
```

Comment:
This should probably be updated/removed too, as just opening this file probably isn't necessary. Also, this file will be changing dramatically whenever #6167 goes in.